### PR TITLE
Make the char gallery more space-efficient

### PR DESCRIPTION
**Pull request by conanak99**
Update the character gallery with nicer HTML/CSS and make it more space-efficient

Before
Screenshot 2023-02-27 at 9 28 05 PM

After
Screenshot 2023-02-27 at 9 51 38 PM

### CODE REVIEW

**Comment by oobabooga** [comment]
That looks amazing!!!

The only issue I found was that the pictures overlap on mobile devices. I have added a commit with a possible solution to this. Does it make sense?

**Comment by conanak99** [comment]
Sorry, I push -f and ignore your commit :D. 

Setting `auto-fit` and `minmax` seems to work, modern CSS is powerful :P.

ezgif com-optimize

File: extensions/gallery/script.py [hunk 1]
```diff
@@ -42,6 +42,7 @@ def generate_html():
         display: block;
         font-size: 1.2rem;
         font-weight: 600;
+        overflow-wrap: anywhere;
```
Comment (conanak99) [inline extensions/gallery/script.py:45]: 👍

**Comment by oobabooga** [comment]
Looks good to me now. Thanks again for this 😃

**Pull request merged**